docs/table-of-contents.md

```diff
@@ -13,6 +13,11 @@ To automatically add a `toc` section to your `myst.yml` file using filenames to
 myst init --write-toc
```

Comment:
This seems to just add all of the base level files. what if we want to define a structure for toctrees or no toctree in each section of the site?

Reply:
The idea is that this just creates the initial structure for you, and you can then modify it however you want. re: `no toctree` could you describe better what the end result looks like that you're shooting for?